IOS/SwiftSeedProject/SwiftSeedProject/Interfaces/ViewControllers/NewsFeedViewController.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  SSNewsFeedViewController.swift
+//  SwiftSeedProject
+//
+//  Created by Lucas Pelizza on 4/18/17.
+//  Copyright © 2017 Making Sense. All rights reserved.
+//
+
+import UIKit
+
+//TODO: this protocol should be a property of VM
+//TODO: we need to remove this code from here
+protocol NewsFeedViewModelDelegate {
+    func updateUI(news: [NewsArticleViewModel])
+    func navigateTo(vc:UIViewController)
+}
+
+final class NewsFeedViewController: UIViewController, NewsFeedViewModelDelegate {
+
+    //TODO: here we should add the VM that Brian is working
+    var viewModel: AnyObject!
+    
+    @IBOutlet weak var cvNews: NewsCollectionView!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        //TODO: here we should init the VM
+        self.cvNews.setup(viewModelDelegate: self.viewModel! as! NewsViewModel)
+    }
+
+    override func didReceiveMemoryWarning() {
+        super.didReceiveMemoryWarning()
+    }
+    
+    //MARK: - SSNewsFeedVMDelegate
+    func updateUI(news: [NewsArticleViewModel]) {
+        cvNews.setItems(news)
+    }
+    
+    func navigateTo(vc:UIViewController) {
```

Comment:
I think this function should be a protected function located in a BaseViewController so is available to every ViewController that extends it.